README.md

```diff
@@ -109,7 +109,40 @@ Beware though, when adding a volatile default value:
 ```ruby
 add_column :users, :created_at, default: 'clock_timestamp()'
 ```
-PG will still needs to update every row of the table, and will most likely statement timeout for big table. In this case, your best bet is to add the column without default, set the default, and backfill existing rows.
+PG will still needs to update every row of the table, and will most likely statement timeout for big table. In this case, safe-pg-migrations can automatically backfill data when the option `default_value_backfill:` is set to `:default_value_backfill`. 
+
+### Default value Backfill
```

Comment:
I would add that in another item (ex: Safe add_column - adding a volatile default value) of the accordion, as it's clearly an edge case

Reply:
and solved